toc/rfc/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Foundry Request For Comments
+
+Cloud Foundry community members use this directory as a forum to make, discuss and record both process and technical decisions. The outcomes of these discussions can be either an Action Plan, a new Standard or Process that the Cloud Foundry community should follow. This repository is open as a reference for all working groups and for the wider community.
+
+## Process
+
+1. Create a new branch on this repo and copy `rfc-000-template.md` to `rfc-000-my-proposal.md` and edit. The numbering used should be the next sequential number that is available when you create your PR.
+1. If your RFC is only relevant for a single Working Group, please place it in a directory for the working group within toc/rfc.
+   1. If your RFC is only relevant for a sigle Working Group, and the Working Group hosts RFCs elsewhere and this step has not already been completed, please create a directory within toc/rfc and place a link to where the RFCs can be found to aid in discoverability and follow the already agreed process. 
+1. Include any images etc in a separate directory named `rfc-000` (using the number of your RFC) and link to them.
+1. Make a Pull Request (PR) for your branch.
+1. Rename your file and directory with the number of the PR and push as a new commits.
+1. Tag `@toc` and any relevant working groups in your PR to enable discussion of the RFC. 
+1. Cloud Foundry community members discuss your proposal using both inline comments against your RFC document and the general PR comments section.
+1. As changes are requested and agreed in comments, make the changes in your RFC and push them as new commits.
+1. Stay active in the discussion and encourage and remind other relevant people to participate. If you’re unsure who should be involved in a discussion, ask your Technical Lead, Execution Lead or the TOC. If you start an RFC it’s up to you to push it through the process and engage people.
+1. Once consensus is reached and approvals given using the Github review system, the PR can be merged.
```

Comment:
This stage might also benefit from a final comment period like the one in the CNB process (https://github.com/buildpacks/rfcs#final-comment-period). That final comment period and the motion to enter it could also be a common process for acceptance and rejection, since both require consensus of the relevant governance body (TOC or WG).